plugins/plugins.cpp

```diff
@@ -2844,6 +2855,33 @@ static void initStatic__sonusmodular()
     }
 }
 
+
+static void initStatic__StarlingVia()
+{
+    Plugin* const p = new Plugin;
+    pluginInstance__StarlingVia = p;
+
+    const StaticPluginLoader spl(p, "StarlingVia");
+    if (spl.ok())
+    {
+#define modelScanner modelStarlingViaScanner
+#define Scale starlingViaScale
```

Comment:
the scale and wavetable macros are not needed here, not sure about it being needed on this file at all tbh

Reply:
hmm, pretty sure I had conflicts when not defining these. can try again without

Reply:
Hmm, commented these out, rebuilt the StarlingVia components and it works fine. So I'll remove.